tests/fib/test_fib.py

```diff
@@ -129,7 +129,28 @@ def setup_hash(self, testbed, duthost, ptfhost):
 
         g_vars['testbed_type'] = testbed['topo']['name']
         g_vars['router_mac'] = duthost.shell('sonic-cfggen -d -v \'DEVICE_METADATA.localhost.mac\'')["stdout_lines"][0].decode("utf-8")
-            
+
+        # generate available send packet ports
```

Comment:
this looks like a good function. can you define a function?

get_router_interface_ports()
"""get all physical ports associated with router interface (physical router interface, port channel router interface and vlan router interface)

Reply:
done.